platform_plugin_aspects/utils.py

```diff
@@ -309,3 +309,46 @@ def get_tags_for_block(usage_key) -> set:
             tag = tag.parent
 
     return list(serialized_tags)
+
+
+def get_user_dashboard_locale(user):
+    """
+
+    Get Superset dashboard locale from a user's preferred language.
+
+    """
+    try:
+        user_language = (
+            get_model("user_preference").get_value(user, "pref-lang") or "en"
```

Comment:
Eventually we would like to make the default language configurable, but in the meantime can we define `en` as a constant so there are fewer places to change it later?